tests/fft.hpp

```diff
@@ -214,3 +216,87 @@ static void test_fft()
     ASSERT_LE(criterion, epsilon);
     ASSERT_LE(criterion2, epsilon);
 }
+
+template <typename ExecSpace, typename MemorySpace, typename Tin, typename Tout, typename X>
+static void test_fft_norm(ddc::FFT_Normalization const norm)
+{
+    DDom<DDim<X>> const x_mesh
+            = DDom<DDim<X>>(ddc::init_discrete_space(DDim<X>::
+                                                             init(ddc::Coordinate<X>(-1. / 4),
+                                                                  ddc::Coordinate<X>(1. / 4),
+                                                                  DVect<DDim<X>>(2))));
+    ddc::Chunk _f = ddc::Chunk(x_mesh, Allocator<MemorySpace, Tin>());
+    ddc::ChunkSpan f = _f.span_view();
+    ddc::for_each(
+            policy<ExecSpace>(),
+            ddc::get_domain<DDim<X>>(f),
+            DDC_LAMBDA(DElem<DDim<X>> const e) { f(e) = static_cast<Tin>(1); });
```

Comment:
what's `DElem` ?

Reply:
uhm... ok, I see. I'm not sure it's a good idea to introduce file-specific aliases, it makes the understanding all the more complicated for the reader.

Reply:
I put as resolved to merge (because not in the scope of current merge request), if it is really important we can speak about it again later. Imo, most users will use aliases like we do in Gysela for big projects (far more readable) so using it in tests too gives them conventions to follow.